packages/web-wallet/src/state/userAccounts/selectors.js

```diff
@@ -0,0 +1,96 @@
+import { createSelector } from 'reselect';
+import { isEmpty, isNil } from 'lodash';
+import { parseBalance, parsedBalanceValue } from '@mobius-network/core';
+
+import {
+  getEntitiesObject,
+  getIsFetching,
+  getIsSuccess,
+  getOperation,
+} from 'state/requests';
+import { getDappCatalog, parseDappCatalogEntry } from 'state/storeAccount';
+
+export const getUserAccount = createSelector(
+  state => getEntitiesObject(state, { entity: 'userAccounts' }),
+  (_, { accountNumber } = {}) => accountNumber,
+  (userAccounts, accountNumber) => {
+    if (isEmpty(userAccounts)) {
+      return {};
+    }
+
+    return userAccounts[accountNumber] || {};
```

Comment:
`lodash.get` would be helpful here too: `return get(userAccounts, 'accountNumber') || {}`

Reply:
Neat suggestion. Thanks